static/js/UserHistoryPanel.jsx

```diff
@@ -77,10 +101,10 @@ UserHistoryPanel.propTypes = {
 };
 
 
+
 const UserHistoryList = ({store, scrollableRef, menuOpen, toggleSignUpModal}) => {
   const [items, setItems] = useState(store.loaded ? store.items : null);
 
-  // Store changes when switching tabs, reset items
```

Comment:
I'd say this comment is actually a non-obvious effect, so might be worth leaving it? (Unless you erased it because it's incorrect 😅 )